controllers/reservation.controllers.js

```diff
@@ -266,7 +266,7 @@ const addReservation = async ({
               },
               label: {
                 type: 'plain_text',
-                text: 'Label',
+                text: 'A que espacio vas?',
```

Comment:
I think "Selecciona un espacio de trabajo" is better. @juanizquierdo19 any thoughts?

Reply:
We were going for an informal tone approach, that's why the 'A que espacio vas?'. If it's too informal, we can go ahead and change it for the suggestion @mauriguz did.

Have in mind, all communication should have the same tone.

Reply:
@MathiasMoralesDixtra Let's leave it as is, we can always make updates later on